### PR TITLE
Added Bulgarian Translation

### PR DESCRIPTION
**Pull request by twlvnn**

### CODE REVIEW

File: po/bg.po [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+msgid "Identify hashing algorithms"
+msgstr "Идентифицирайте алгоритми за хеширане"
+
+#: data/io.github.zefr0x.hashes.desktop.in:7
+msgid "Hash Identifier"
+msgstr "Хеш Идентификатор"
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;detect;identify;хеш;засичане;идентифициране;"
```
Comment (zefr0x) [inline po/bg.po]: This should not include English key words, just Bulgarian as possible: https://github.com/zefr0x/hashes/pull/6#discussion_r1548860728

File: po/bg.po [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+"бил използван за даден низ."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:10
+msgid "Features:"
+msgstr "Функции:"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:12
+msgid "📺 Popularity Ratings - Most popular hashes showed first."
+msgstr ""
+"Класификация на популярността - Най-популярните хешове се показват първи."
```
Comment (zefr0x) [inline po/bg.po]: looks like you missed the emoji here

**Review by zefr0x — CHANGES_REQUESTED** [review]

**Comment by twlvnn** [comment]
Thank you for the feedback zefr0x, I've corrected the mistakes and fixed another mistake I made with the hyphenation. Let me know if there's anything I missed!

**Comment by zefr0x** [comment]
Thanks to you, Bulgarians can use the application in their native language

**Pull request merged**